### PR TITLE
Chore/post comment delete

### PR DESCRIPTION
**Pull request by karljerico**
### OVERVIEW

- Used the JS and modal for post and comment delete
- Rename the javascript file
- Moved the delete modal to modal folder
- Edit UI modal

### SNIPPETS
image

### CODE REVIEW

**Review by hipe01mark — DISMISSED** [review]
LGTM 🙇

**Review by nadj1233 — DISMISSED** [review]
LGTM 💯

File: src/public/js/deleteModal.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+        modal.find('.confirm-delete').data('id', deleteId);
+        modal.find('.confirm-delete').data('type', deleteType);
+
+        modal.modal('show');
+
+        modal.find('.confirm-delete').click(function () {
+            const csrfToken = $('meta[name="csrf-token"]').attr('content');
+            $.ajax({
+                url: '/' + deleteType + 's/' + deleteId,
```
Comment (hipe01mark) [inline src/public/js/deleteModal.js]: I noticed that you guys always using manual concatenation. You can use ECMAScript Concatenation.

```javascript
//Old way
url: '/' + deleteType + 's/' + deleteId,
 
//New Way
url: `/${deleteType}s/${deleteId}`
```

File: src/public/js/deleteModal.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                type: 'POST',
+                data: {
+                    '_method': 'DELETE',
+                    _token: csrfToken,
+                },
+                success: function () {
+                    $('.delete-button[data-id="' + deleteId + '"]').closest('.card').remove();
```
Comment (hipe01mark) [inline src/public/js/deleteModal.js]: I noticed that you guys always using manual concatenation. You can use ECMAScript Concatenation.

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

**Pull request merged**